### ISSUE TITLE
The (max 3) wording

### ISSUE THREAD

**Issue by DemiurgeJ** [mlenser/kryx-rpg-issues#639]
I first noticed this when looking over the Acolyte, but I see that it is used elsewhere too. To explain the issue, I will use the Priest's Armour of Faith ability as an example. 

The "max 3" at the end of it (your Defense equals 9 + half your proficiency bonus + your Dexterity + your spellcasting ability (max 3)) could indicate the total bonus could be +3 to a maximum of AC 12, or a maximum of +3 from spellcasting ability. I suggest rewording it to "your Defense equals 9 + half your proficiency bonus + your Dexterity + up to +3 of your spellcasting ability," and a more universal change to other abilities that use the (max X) ruling.

Its isn't a big issue, or one that _needs_ to change, but it would help for niche clarifications sake.

**Comment by mlenser** [comment]
The current formatting is meant to match the book's formatting 
![image](https://user-images.githubusercontent.com/235857/102405912-190b4c00-3fea-11eb-808f-5116557af41e.png)

It also matches some other similar wording cases
![image](https://user-images.githubusercontent.com/235857/102406135-73a4a800-3fea-11eb-9735-a71085ea1ba3.png)

The wording is
> your Defense equals 9 + half your proficiency bonus + your Dexterity + your spellcasting ability (max 3).

Max only applies to the last part and it is used quite commonly in the system.
I think `+ your spellcasting ability (max 3)` is better wording than `+ up to +3 of your spellcasting ability`
